packages/runtime-dom/__tests__/customElement.spec.ts

```diff
@@ -1131,6 +1131,92 @@ describe('defineCustomElement', () => {
       expect(target.innerHTML).toBe(`<span>default</span>`)
       app.unmount()
     })
+
+    test('toggle nested custom element with shadowRoot: false', async () => {
```

Comment:
Hi @edison1105 , thanks for the fix, it works fine in our app as well!

I'm not sure if it is related to the current issue or I need to create a separate one, but If in custom element slot we have a `v-if` clause, then it never renders even if the clause is true, but if it's `v-show` - it works fine.

Reply:
Do you want me to add a test case to this branch or I can create an issue.

Reply:
Thank you so much for this @edison1105 
The issue is created here https://github.com/vuejs/core/issues/13206